### PR TITLE
404 and feedback

### LINKED ISSUE #2: To Do: Feedback and bug report form

### LINKED ISSUE #7: To Do: 404 Page

### PR DESCRIPTION
**Pull request by rengzhengcodes**
Implemented and solved the 404.html and Feedback.html todos.

### CODE REVIEW

File: feedback.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			<a href="404.html">SCHOLARSHIPS</a>
+			<a href="404.html">ACADEMICS</a>
+		</nav>
+		<main>
+			<h1>Website Feedback and Bug Reports</h1>
+			<div class="form">
+				<iframe src="https://docs.google.com/forms/d/e/1FAIpQLScvnE8I_Nhff12qQqvm8Yu0tZrCNBrZiA6QCS4THa7ooJetoA/viewform?embedded=true" width="640" height="1087" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+			</div>
+		<footer>
+			Last Updated: 2020/12/17 | Junior Caucus IT | <a href="feedback.html">Bug Reports and Feedback</a>
```
Comment (Clue88) [inline feedback.html]:
```suggestion
			Last Updated: 2020/12/18 | Junior Caucus IT | <a href="feedback.html">Bug Reports and Feedback</a>
```

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<html>
+	<head>
+		<title>Stuyvesant High School Junior Caucus</title>
+		<link rel="stylesheet" href="index.css" type="text/css"/>
+	</head>
+	<body>
+		<nav>
+			<a href="index.html">HOME</a>
+			<a href="about.html">ABOUT US</a>
+			<a href="404.html">OPPERTUNITIES</a>
```
Comment (willowiscool) [inline 404.html]:
```suggestion
			<a href="404.html">OPPORTUNITIES</a>
```

File: feedback.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<html>
+	<head>
+		<title>Stuyvesant High School Junior Caucus</title>
+		<link rel="stylesheet" href="index.css" type="text/css"/>
+	</head>
+	<body>
+		<nav>
+			<a href="index.html">HOME</a>
+			<a href="about.html">ABOUT US</a>
+			<a href="404.html">OPPERTUNITIES</a>
```
Comment (willowiscool) [inline feedback.html]:
```suggestion
			<a href="404.html">OPPORTUNITIES</a>
```

File: 404.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+			<a href="404.html">SCHOLARSHIPS</a>
+			<a href="404.html">ACADEMICS</a>
+		</nav>
+		<main>
+			<h1>404 Page Not Found</h1>
+			<h2>If you followed a link on our page to get here, the page you're looking for may be under construction. Check back soon!</h2>
+		<footer>
+			Last Updated: 2020/12/17 | Junior Caucus IT | <a href="feedback.html">Bug Reports and Feedback</a>
```
Comment (willowiscool) [inline 404.html]:
```suggestion
			Last Updated: 2020/12/18 | Junior Caucus IT | <a href="feedback.html">Bug Reports and Feedback</a>
```

File: index.html [hunk 5]
```diff
@@ -6,11 +6,11 @@
...
 	<body>
 		<nav>
-			<a href="index.html">HOME</a>
-			<a href="about_us.html">ABOUT US</a>
-			<a href="#">OPPORTUNITIES</a>
-			<a href="#">SCHOLARSHIPS</a>
-			<a href="#">ACADEMICS</a>
+      <a href="index.html">HOME</a>
+			<a href="about.html">ABOUT US</a>
+			<a href="404.html">OPPERTUNITIES</a>
```
Comment (willowiscool) [inline index.html]:
```suggestion
			<a href="404.html">OPPORTUNITIES</a>
```

File: index.html [hunk 6]
```diff
@@ -31,7 +31,7 @@ <h2>December 10th, 2020</h2>
 				</div>
 			</div>
 		<footer>
-			This is the footer
+			Last Updated: 2020/12/17 | Junior Caucus IT | <a href="feedback.html">Bug Reports and Feedback</a>
```
Comment (willowiscool) [inline index.html]:
```suggestion
			Last Updated: 2020/12/18 | Junior Caucus IT | <a href="feedback.html">Bug Reports and Feedback</a>
```

**Review by willowiscool — COMMENTED** [review]
just trying to see how reviewing works

**Review by Clue88 — DISMISSED** [review]

**Pull request merged**